roles/openshift_metrics/tasks/install_hawkular_schema_job.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# We cannot use oc apply here because the Job template has immutable fields
+# on which oc apply will fail.
+- name: remove hawkular-metrics-schema job
+  command: >
+    {{ openshift_client_binary }} -n {{ openshift_metrics_project }} --config={{ mktemp.stdout }}/admin.kubeconfig
+    delete job hawkular-metrics-schema
+  register: delete_schema_job
+  ignore_errors: True
```

Comment:
isn't there a flag we can use on the command instead to not fail if not found or will that make the `changed_when` not work as expected?

The reason I'd rather not use `ignore_errors: True` is because this will show an error in the playbook output (even though the task wont be marked as failed)

Reply:
@ewolinetz PR updates and commits squashed.